actors/builtin/power/power_state.go

```diff
@@ -28,8 +28,8 @@ type State struct {
 	// A queue of events to be triggered by cron, indexed by epoch.
 	CronEventQueue cid.Cid // Multimap, (HAMT[ChainEpoch]AMT[CronEvent]
 
-	// Last chain epoch OnEpochTickEnd was called on
-	LastEpochTick abi.ChainEpoch
+	// First epoch in which a cron task may be stored.
```

Comment:
Please note here that this scheme means the actor will iterate every epoch from FirstCronEpoch up to the current epoch, checking the CronEventQueue.